std/os.zig

```diff
@@ -133,6 +133,11 @@ fn getRandomBytesDevURandom(buf: []u8) !void {
     const fd = try openC(c"/dev/urandom", O_RDONLY | O_CLOEXEC, 0);
     defer close(fd);
 
+    const st = try fstat(fd);
+    if (!system.S_ISCHR(st.mode)) {
```

Comment:
S_ISCHR should be in the bits.zig files, and std/os.zig is one of the few places where we use `usingnamespace` on all the system bits. In this particular file (but not others) the convention is to omit the namespace.
```suggestion
    if (!S_ISCHR(st.mode)) {
```